cyclonedx/model/bom.py

```diff
@@ -80,6 +82,30 @@ def timestamp(self) -> datetime:
     def timestamp(self, timestamp: datetime) -> None:
         self._timestamp = timestamp
 
+    @property
+    def component(self) -> Optional[Component]:
+        """
+        Get a Component of this Bom Metadata.
+
+        Returns:
+            `cyclonedx.model.component.Component` instance for this Bom Metadata.
+        """
+        return self._component
+
+    @component.setter
+    def component(self, component: Component) -> None:
+        """
+        Set a Component for this Bom Metadata.
```

Comment:
@artem-smotrakov - I've tried to mirror the descriptions provided in the Specification here - so ideally this doc would read:

> The (optional) component that the BOM describes.